### PR TITLE
docs: ✏️ Add Hyperledger Besu examples

### PR DESCRIPTION
**Pull request by atoulme**
Add examples showing how to monitor Hyperledger Besu during a fast or a full sync.

The examples consist of a docker-compose set up and a Splunk application.

### CODE REVIEW

**Review by Moogz — APPROVED** [review]

File: examples/besu-sync/full-sync/besu-full-sync/metadata/local.meta [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[app/ui]
```
Comment (ziegfried) [inline examples/besu-sync/full-sync/besu-full-sync/metadata/local.meta:1]: same nit: let's omit this file

**Review by ziegfried — APPROVED** [review]
Small nitpick, otherwise looks good

**Comment by atoulme** [comment]
OK, removed both files, thanks! I'll merge when all tests pass.

**Pull request merged**